src/common/log.js

```diff
@@ -11,30 +12,39 @@ const logger = bunyan.createLogger({
   process: process.type
 })
 
+function logToStdout() {
+  logger.addStream({
+    stream: process.stdout,
+    level: 'debug'
+  })
+}
 
-function init(dir) {
+function logToFolder(dir) {
+  if (dir) {
+    let logDir = join(dir, 'log')
```

Comment:
Let's assume that the full directory path is passed-in. I think it makes it less flexible if we always add 'log' here.